components/prism-docker.js

```diff
@@ -0,0 +1,8 @@
+Prism.languages.docker = {
+	'keyword': {
+    pattern: /^\s*\b(ONBUILD|FROM|MAINTAINER|RUN|EXPOSE|ENV|ADD|COPY|VOLUME|USER|WORKDIR|CMD|ENTRYPOINT)\b\s/mi
```

Comment:
The `\b` parts seems redundant with the `\s` parts (the latter implies the former). Also, do we really need to include those whitespaces in the match? What about something like:

```javascript
{
	pattern: /(^\s*)(?:ONBUILD|...|ENTRYPOINT)(?=\s)/mi,
	lookbehind: true
}
```

Finally, what about the [`LABEL` keyword](https://docs.docker.com/reference/builder/#label)?